src/locales/i18n.en.json

```diff
@@ -140,10 +140,10 @@
         "image": "Passport image"
       },
       "driverLicense": {
-        "title": "Driver license",
```

Comment:
also need to change `constants.documentType.driverLicense` value

Reply:
Fixed